x-pack/plugins/reporting/export_types/common/execute_job/omit_blacklisted_headers.ts

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { omit } from 'lodash';
+import { KBN_SCREENSHOT_HEADER_BLACKLIST } from '../../../common/constants';
+import { KbnServer, ReportingJob } from '../../../types';
+
+export const omitBlacklistedHeaders = ({
+  job,
```

Comment:
I could go either way on this, but I think it would be better to not have the `job` param passed into this file. It seems like it's here for a convenience for a `mergeMap`, but that's an outside detail that I think this file doesn't need to be concerned with.

Reply:
Job is a required parameter of one of the future functions in the pipe command and if its not returned here the function that requires it will throw and exception. I started looking into removing rxjs and change these function to simple calls with input parameters and return values but that is for a future PR.